app/models/manageiq/providers/base_manager/refresher.rb

```diff
@@ -251,7 +251,8 @@ def preprocess_targets_manager_refresh
             if ems.allow_targeted_refresh?
               # We can disable targeted refresh with a setting, then we will just do full ems refresh on any event
               ems_event_collection = InventoryRefresh::TargetCollection.new(:targets    => sub_ems_targets,
-                                                                            :manager_id => ems_id)
+                                                                            :manager_id => ems_id,
```

Comment:
do we still need to set the `manager_id`?

Reply:
`InventoryRefresh::TargetCollection` does use both `@manager` and `@manager_id` so IMO it would be best to set both.

It is able to handle if `@manager` is set but `@manager_id` is not, but it doesn't handle that in the `#initialize` method so right now it seems safer to just set both.